#pragma once


#include "../pb_msg/core/rpc_msg.pb.h"
#include "../pb_msg/core/pubsub.pb.h"
#include "../pb_msg/core/nats_msg.pb.h"
#include "../pb_msg/core/mysql_proxy_msg.pb.h"
#include "../pb_msg/core/opcodes.pb.h"
#include "../pb_msg/core/common.pb.h"
#include "../pb_msg/core/service_discovery.pb.h"
#include "../pb_msg/core/route_register.pb.h"